src/main/java/edu/harvard/iq/dataverse/engine/command/impl/MoveDatasetCommand.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Should probably remove these default headers.

Reply:
Yeah, I'm not a fan of those.